roles/load-ycsb/templates/workload.yml.j2

```diff
@@ -0,0 +1,52 @@
+---
+kind: Job
+apiVersion: batch/v1
+metadata:
+  name: 'ycsb-data-load-job-{{ ycsb_workload }}-{{ trunc_uuid }}'
+  namespace: '{{ operator_namespace }}'
+spec:
+  template:
+    metadata:
+      name: '{{ meta.name }}-ycsb-data-load'
+      namespace: '{{ operator_namespace }}'
+      labels:
+        name: 'ycsb-load-{{ trunc_uuid }}'
+    spec:
+      containers:
+      - name: ycsb
+        image: "quay.io/cloud-bulldozer/ycsb-server:latest"
+        imagePullPolicy: Always
+        env:
+        - name: clustername
+          value: "{{ clustername }}"
+        - name: es
+          value: "{{ elasticsearch.server | default() }}"
+        - name: es_port
+          value: "{{ elasticsearch.port | default() }}"
+        - name: es_index
+          value: "{{ es_index }}"
+        - name: user
+          value: "{{ test_user | default() }}"
+        - name: workload
+          value: "{{ ycsb_workload }}"
+        - name: num_records
+          value: "{{ ycsb.recordcount }}"
+        - name: num_operations
+          value: "{{ ycsb.operationcount }}"
+        - name: uuid
+          value: "{{ uuid }}"
+        command: ["/bin/sh", "-c"]
+        args: ["python /opt/snafu/run_snafu.py --tool ycsb -r 1 -l -d {{ ycsb.driver }} -u {{ uuid }} --user {{ test_user | default("ripsaw") }} -w {{ ycsb_workload }} -x '{{ ycsb.options_load }}'"]
```

Comment:
@aakarshg swapping "->' in the template should fix your "ing issue that you had earlier as well as require no change to existing cr files. I do the same thing in https://github.com/cloud-bulldozer/ripsaw/pull/339

```
        args: ['python /opt/snafu/run_snafu.py --tool ycsb -r 1 -l -d {{ workload_args.driver }} -u {{ uuid }} --user {{ test_user | default("ripsaw") }} -w {{ ycsb_workload_load }} -x "{{ workload_args.options_load }}"']
```